include/variant_detection/variant_output.hpp

```diff
@@ -21,13 +22,15 @@
  *          The quality of an SV is estimated based on the size of the cluster (i.e. the number of reads supporting the SV).
  */
 void find_and_output_variants(std::vector<Cluster> const & clusters,
+                              std::set<std::tuple<std::string, int32_t>> & sn_ln_tag_tuple_set,
                               cmd_arguments const & args,
                               std::ostream & out_stream);
```

Comment:
I would either put 'out' parameter at the beginning or at the end. Since you already have them at the beginning of the previous function, maybe move it to the first place. (doc also needs an update then)

```suggestion
void find_and_output_variants(std::set<std::tuple<std::string, int32_t>> & sn_ln_tag_tuple_set,
                              std::vector<Cluster> const & clusters,
                              cmd_arguments const & args,
                              std::ostream & out_stream);
```